modules/expandable-panel/ExpandablePanel.tsx

```diff
@@ -0,0 +1,229 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState
+} from "react";
+import { withStyles } from "@diana-ui/base";
+import {
+  StandardProps,
+  WithStylesProps,
+  ThemeStyleSheetFactory,
+  Theme
+} from "@diana-ui/types";
+import { useWindowSize } from "@diana-ui/hooks";
+import { Icon } from "@diana-ui/icon";
+import { TextHighlight } from "@diana-ui/typography";
+
+const SLIDE_ANIMATION_DURATION_MS = 200;
+
+// @ts-ignore
```

Comment:
why is this needed ?

Reply:
Since the standard div props are being extended, I can't override onClick with a different signature without TS complaining 😕